pipenv/core.py

```diff
@@ -483,11 +483,11 @@ def abort():
         # about, so confirm first.
         if "VIRTUAL_ENV" in os.environ:
             if not (
-                project.s.PIPENV_YES or click.confirm("Remove existing virtualenv?", default=True)
+                project.s.PIPENV_YES or click.confirm("Use existing virtualenv?", default=True)
             ):
                 abort()
         click.echo(
-            crayons.normal(fix_utf8("Removing existing virtualenv..."), bold=True), err=True
+            crayons.normal(fix_utf8("Using existing virtualenv..."), bold=True), err=True
```

Comment:
There is no longer need for `fix_utf-8` since pipenv does not support python2, this can be removed.
Also, `click.echo` does not need crayons.normal. IMHO  both can be removed.

Reply:
I didn't change this and would rather do it in another commit because in core.py alone the same thing occurs more than 30 times.